source/main/cardano/CardanoWalletLauncher.js

```diff
@@ -43,15 +43,20 @@ export async function CardanoWalletLauncher(
       break;
   }
 
-  const walletStdio: string[] = ['inherit', logStream, logStream, 'ipc'];
+  const walletStdio: string[] = ['pipe', 'pipe', 'pipe', 'ipc'];
```

Comment:
what happens if you just leave this line as-is, i believe its already the right type to work here

Reply:
It should have been, as the return type of the library was an extension of `WriteStream`, but when passed to `stdio` it threw an invalid type error, saying that it was not a valid Stream.

I couldn't see any difference in the CPU profile when using pipe, so I think its okay.

Reply:
And to confirm, traditionally with a Stream you wait for the `on` event as that represents when the fd is opened, at which point it is valid to be passed to `stdio`. However that didn't help, and `on` was called every time the log was cycled, which was super annoying as then we tried to launch new wallets and nodes each log cycle!

So this change actually solves more than one thing